CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ set(
 # Primary user facing options
 option(MANIFOLD_CROSS_SECTION "Build CrossSection for 2D support" ON)
 option(MANIFOLD_DEBUG "Enable debug tracing/timing" OFF)
+option(MANIFOLD_ASSERT "Enable assertions - requires MANIFOLD_DEBUG" OFF)
```

Comment:
I remember there is a fancy dependent option thing that may be useful here. At least we may want to issue an error if the user only enabled `MANIFOLD_ASSERT` without enabling `MANIFOLD_DEBUG`?

Reply:
Good idea - want to throw that into a follow-up PR?